server/rest.py

```diff
@@ -119,12 +121,10 @@ def createTiles(self, item, params):
                 # a file ID.
                 raise RestException('"fileId" must be a file on the same item'
                                     'as "itemId".')
-
-            if largeImageFile['mimeType'] == 'image/tiff':
-                # TODO: we should ensure that it is a multiresolution tiled TIFF
+            try:
+                TiffGirderTileSource(None, largeImageFile['_id'])
```

Comment:
Why not just do:
```
try:
    item['largeImage'] = largeImageFile['_id']
except TileSourceException:
    del item['largeImage']
    ... # create job
```
That way, we can avoid the whole extra parameter. It's not a big deal if this gets added to the cache, as it will never be used again and eventually flushed.